package.json

```diff
@@ -1,12 +1,11 @@
 {
   "scripts": {
-    "test": "xo && ava"
+    "test": "xo && ava",
+    "release": "webstore upload --source=extension --auto-publish"
```

Comment:
Makes sense - https://github.com/DrewML/chrome-webstore-upload-cli/issues/3

Will make the update in the next day or two and then update this PR